ledger/error/src/main/scala/com/daml/error/generator/DocItem.scala

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.error.generator
+
+import com.daml.error.{Explanation, Resolution}
+
+/** Contains error presentation data to be used for documentation rendering on the website.
+  *
+  * @param className The error class name (see [[com.daml.error.ErrorCode]]).
```

Comment:
Error code or error class? 
`className` vs `(see [[com.daml.error.ErrorCode]]).`

Reply:
Looking at the usages in Canton I now see that this field refers to ~`error.getClass.getName`

Reply:
My suggestions:
- rename it to `errorCodeName`.
- add `name` attribute to `ErrorCode` implemented as `this.getClass.getName`

Reply:
That's a good suggestion, but I would refrain from refactoring in this PR. I'll add a TODO, though